upgrading/automated_upgrades.adoc

```diff
@@ -702,6 +702,10 @@ This will cause the GlusterFS pod to terminate on that node.
 $ oc label node <node_name> <daemonset_label>
 ----
 
+.. Add an additional label (for example, `type=upgrade`) to the node you want to upgrade.
+
+.. Run the upgrade playbook with `-e openshift_upgrade_nodes_label="type=upgrade"`
+
```

Comment:
I think we need to change the order here a bit:
- first remove daemonset label  
- then add label to node, e.g. type=upgrade
- after that run the upgrade playbook with  -e openshift_upgrade_nodes_label="type=upgrade"
otherwise we would first upgrade and then add the label which would not do wat we want: updating one, and only one node with that label

So the following would make sense to me

``
    Run oc get daemonset to verify the label found under NODE-SELECTOR. The default value is storagenode=glusterfs.  _(I think that needs to be run in the project where gluster pods are running)_

    Remove the daemonset label from the node:

    $ oc label node <node_name> storagenode-

    This will cause the GlusterFS pod to terminate on that node.

    Add an additional label (for example, type=upgrade) to the node you want to upgrade.

    To run the upgrade playbook on the single node where you terminated GlusterFS use -e openshift_upgrade_nodes_label="type=upgrade"

    When the upgrade completes, relabel the node with the daemonset selector:

    $ oc label node <node_name> <daemonset_label>

    Wait for the GlusterFS pod to respawn and appear.

    oc rsh into the pod and verify all volumes are healed:

    $ oc rsh <GlusterFS_pod_name>
    $ for vol in `gluster volume list`; do gluster volume heal $vol info; done

    Ensure all of the volumes are healed and there are no outstanding tasks. The heal info command lists all pending entries for a given volume’s heal process. A volume is considered healed when Number of entries for that volume is 0.

    Remove the node label (for example, type=upgrade) and go to the next GlusterFS node.

``

Reply:
Agreed. I would also change `$ oc label node <node_name> storagenode-` to `$ oc label node <node_name> <daemonset_label>-` since the "storagenode" part can be overridden.